tensorboard/webapp/feature_flag/types.ts

```diff
@@ -52,4 +52,6 @@ export interface FeatureFlags {
   enableSuggestedCards: boolean;
   // Persists pinned scalar cards across multiple experiments.
   enableGlobalPins: boolean;
+  // Adds a new mode of coloring, by matching the regexp of experiment name.
+  enableColorByExperiment: boolean;
```

Comment:
QQ. So the purpose of enableColorByExperiment is to 1) protect this feature until release and 2) block the feature in 3p? So won't this feature flag be removed when the feature is released?

Reply:
Yes it will serve both purposes. As for the release in 1P I am planning to just reenable it in 1P code.